web/src/components/feature.js

```diff
@@ -48,7 +48,7 @@ const StyledP = styled.p`
 export default function Feature({ bgColor, content, image }) {
     return (
         <GridWrapper bgColor={bgColor}>
-            <div className="">
+            <div className="text-wrap">
                 <StyledHeading>{content.heading}</StyledHeading>
                 <StyledP>{content.body}</StyledP>
```

Comment:
Should set a max width on the body.

html
![Screen Shot 2021-03-19 at 4 06 01 PM](https://user-images.githubusercontent.com/11489171/111836702-1b24c580-88cd-11eb-8a74-c915ab8a971d.png)

designs
![Screen Shot 2021-03-19 at 4 06 09 PM](https://user-images.githubusercontent.com/11489171/111836722-22e46a00-88cd-11eb-9bd8-5093c6754f88.png)